courses/machine_learning/deepdive2/how_google_does_ml/solutions/analyze_with_bigquery_solution.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<ul><li>Reframe the "Scenario" to be a ML scenario (preparing a ml dataset to train a ml model) - see same comment above.</li><li>Explain/describe what the raw data is about, and what operations need to be performed on it to obtain a training set for a ml model (i.e., how to extract the features and labels)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>